Other/FarmerJoeOutfit.cs

```diff
@@ -133,41 +130,41 @@ public void StartingTheAcc()
             Farm.IcestormArena(20, true);
         }
 
-        InvEn.EnhanceInventory();
-        Farm.IcestormArena(30, true);
+        if (Bot.Player.Level < 30)
+        {
+            InvEn.EnhanceInventory();
+            Farm.IcestormArena(30, true);
+        }
 
         #endregion starting out the acc
 
-
-        #region Obtain the Arcane Blade of Glory / Shadow Blade of Dispair (+20% xp)
+        #region Obtain the Enchanted Victory Blade
+        //Arcane Blade of Glory / Shadow Blade of Dispair (+20% xp)
         Core.Logger("Arcane Blade of Glory / Shadow Blade of Dispair (+20% xp)");
         EVBW.GetWeapon(VictoryBladeStyles.Smart);
-        if (Core.CheckInventory("Arcane Blade of Glory"))
-            Core.ToBank("Arcane Blade of Glory");
-        if (Core.CheckInventory("Shadow Blade of Dispair"))
-            Core.ToBank("Shadow Blade of Dispair");
-        if (Core.CheckInventory("Arcane Blade of Glory"))
-            Core.Equip("Arcane Blade of Glory");
+
```

Comment:
enhanced victory blade is turned into one of these 2, so its an either or situation